include/api/CDataSummarizationJsonSerializer.h

```diff
@@ -23,24 +23,32 @@ class CDataFrame;
 }
 namespace api {
 
-//! \brief Class serializes and deserializes data summarization using chunked JSON blob.
-//! Data summarization contains data rows as well as categorical encoding information.
-// TODO #1849 chunking is not supported yet.
-class API_EXPORT CDataSummarizationJsonSerializer final
-    : public CSerializableToJsonDocumentCompressed {
+//! \brief Class which writes the data summarization as a compressed, base64 encoded
+//! and chunked JSON blob.
+//!
+//! DESCRIPTION:\n
+//! The data summarization contains data rows as well as the feature value encoding
+//! information.
+class API_EXPORT CDataSummarizationJsonWriter final : public CSerializableToJsonDocumentCompressed {
```

Comment:
The class has been renamed but not the file or the include guards, so now there's a mismatch.

Reply:
This header previously contained two classes one for serialisation and one for deserialisation so there was already a mismatch here really. I think we should probably rename and maybe even split out the writing and reading class into separate files. The reason I didn't make this change in this PR is that I factored this out of fixing #1869 and don't want to make merging the two PRs a pain. I therefore propose to postpone this to that PR.